loaders/src/glTF/2.0/Extensions/meshopt_decoder.module.d.ts

```diff
@@ -0,0 +1,11 @@
+// This file is part of meshoptimizer library and is distributed under the terms of MIT License.
```

Comment:
External files should sit in `dist/preview release`, perhaps under a sub directory.

Reply:
Ah, I should have looked at where Draco decoder goes.
I also see similar files in `dist/`. Is the idea that the contents of `dist/preview release` will be copied to `dist` at some point so I should just add the files to `dist/preview release`, not to both?

Reply:
... actually, if I put the file to `dist/`, does this mean I can't just import the file from the .ts extension, and would have to resort to using a URL-based configuration, sourcing the file from `preview.babylonjs.com/meshopt_decoder.module.js`? I was hoping to be able to just import the module and let webpack figure it out from there...

Reply:
>  Is the idea that the contents of dist/preview release will be copied to dist at some point

Yup, `dist/preview release` will be copied to `dist` when we release.

> would have to resort to using a URL-based configuration

I believe so, yes. Ideally, we want this to be configurable anyways, so that people can host their own versions.